wordlist.txt

```diff
@@ -406,6 +406,7 @@ SNARKify
 socio
 solvm
 SPHINCS
+Sproul
```

Comment:
awh, feels good to be part of a spellcheck list :smiling_face_with_tear:

Reply:
🫡 immortalized